core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -439,4 +446,13 @@ public enum SqlConformanceEnum implements SqlConformance {
       return false;
     }
   }
+
+  @Override public boolean allowArrayFunction() {
+    switch (this) {
```

Comment:
`return this == SPARK`

Reply:
I'm following the same coding style as the rest of the file.

https://github.com/apache/calcite/blob/8d502cf421441a791db38ff8c87abc47fa45dd74/core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java#L441-L448

if we want to append to the list, the git diff will look better.